roc/receiver.go

```diff
@@ -185,19 +186,57 @@ func OpenReceiver(context *Context, config ReceiverConfig) (*Receiver, error) {
 		return nil, errors.New("context is closed")
 	}
 
+	var (
+		cTargetLatency           ulonglong
+		cMaxLatencyOverrun       ulonglong
+		cMaxLatencyUnderrun      ulonglong
+		cBreakageDetectionWindow ulonglong
+	)
+
+	convD := []struct {
+		name string
+		goD  time.Duration
+		cD   *ulonglong
+	}{
```

Comment:
This temporary array and for-loop looks like over-complication for this simple piece of code.

Let's just use separate local variables, similar to cPacketLength in OpenSender().